engine/src/main/scala/cromwell/engine/workflow/workflowstore/InMemoryWorkflowStoreDatabase.scala

```diff
@@ -0,0 +1,44 @@
+package cromwell.engine.workflow.workflowstore
+
+import cromwell.core.{WorkflowId, WorkflowSourceFiles}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scalaz.NonEmptyList
+
+class InMemoryWorkflowStoreDatabase extends WorkflowStoreDatabase {
```

Comment:
Is this intended to stand the test of time, or does this also go away when the DB form shows up?